src/Files.App/Helpers/SystemBackdropHelper.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Microsoft.UI.Composition;
+using Microsoft.UI.Composition.SystemBackdrops;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Files.App.Helpers
+{
+	internal sealed class AppSystemBackdrop : SystemBackdrop
+	{
+		private bool isSecondaryWindow;
+		private IUserSettingsService userSettingsService;
```

Comment:
Inject IUserSettingsService here

Reply:
Is there a good reason to? This isn't a service and it'll just be moving the same code elsewhere

Reply:
Well, there are some problems with the current approach:
1. You haven't marked `userSettingsService` as nullable (anything that might access it will result in NRE)
2. The service will be injected every time the method `OnTargetConnected()` is called

Reply:
1. Good point, I'll fix that
2. AppSystemBackdrop doesn't support being used for more than one target so it'll only ever only be called once (I need to add a check for that, thanks for reminding me)

Reply:
@d2dyno1